contrib/ohi-release-notes/run.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+set -euo pipefail
+
+RT_PKG="github.com/newrelic/release-toolkit@latest"
+ARGS="$*"
+
+
+# creating a temporary folder where to build the rt binary and cleaning after exiting.
+TEMP_BIN=$(mktemp -dt release-toolkit-XXX)
+function cleanup() {
+    rm -rf $TEMP_BIN    || true
+    rm "${GIT_ROOT}/CHANGELOG.md.bak" || true
+    rm "${GIT_ROOT}/changelog.yaml"   || true
+}
+trap cleanup EXIT
+
+
+# usage and helm command. It is also an error exit in case flags are not correct.
```

Comment:
helm -> help

Reply:
I forgot how to English and my fingers write things without thinking... Fixed in [#345ad7a](https://github.com/newrelic/release-toolkit/pull/135/commits/345ad7a05cd21e2d2aaf0f3f46b423161917de2e)

Good catch!